scripts/chang_detection_with_lstm.py

```diff
@@ -0,0 +1,408 @@
+#!/usr/bin/env python
```

Comment:
Can you move this file to `pyannote/audio/applications` and rename it to `change_detection.py`?
Then, you need to update `setup.py` `entry_points` section to link to this file.

Generally speaking, try to follow as much as possible the structure and reuse as much code from `pyannote/audio/applications/speech_detection.py` as possible

Also, can you make sure that the different modes are similar to those used in `pyannote/audio/applications/speech_detection.py` (train / validate / tune / apply) 
 
I guess `segment` mode should be renamed to `apply`?
But I do not know what `compare` is about...

I will check the rest of this file once these changes are done.

Very good job, by the way :-)